JSDemos/Demos/Lookup/Basics/description.md

```diff
@@ -1 +1,20 @@
-The Lookup is&nbsp;a&nbsp;UI component that allows an&nbsp;end user to&nbsp;search for an&nbsp;item in&nbsp;a&nbsp;collection shown in&nbsp;a&nbsp;drop-down menu. This demo shows how to&nbsp;create a&nbsp;simple and a&nbsp;grouped Lookup.
\ No newline at end of file
+The Lookup allows users to search through its drop-down list and select a single item from it. The drop-down list can be simple or grouped by a single category.
+
+### Use a simple drop-down list
+To bind the Lookup to a simple list, use the [items[]](/Documentation/ApiReference/UI_Components/dxLookup/Configuration/items/) property. The data array should include strings only.
```

Comment:
Why `strings only`? Any object can be also passed to this property (but we should specify `valueExpr` and `displayExpr` as well)